src/main/webapp/workspace/workspace.js

```diff
@@ -20,22 +11,134 @@ function getFirebaseRef() {
   }
 }
 
+function scrollTabs(event) {
+  // Only translate vertical scrolling to horizontal scrolling.
+  if (event.deltaY) {
+    this.scrollLeft += (event.deltaY);
+    event.preventDefault();
+  }
+}
+
+function switchTab(tab) {
```

Comment:
Add a brief comment explaining what function is doing

Reply:
Added descriptions.